### ISSUE TITLE
"New page" modal window is not localised

### ISSUE THREAD

**Issue by madodig** [Requarks/wiki#2439]
**Describe the bug**
When selecting "New Page" action, modal window that is shown is not shown in Base locale. Strings (such as "Virtual Folders", "Select New Page Location", "Pages", "Cancel", "Select" and "new-page") are hard-coded in assets/js/ui-extra.js.

**To Reproduce**
1. Change Base Locale under Administration / Locale / Locale Settings to a language other than English and click Apply;
2. Navigate to site content;
3. Select "New Page" action;
4. Modal window that is shown is not localised.

**Expected behavior**
Modal window that is shown after "New Page" action is selected, is shown in selected Base locale.

To fix the issue, please:
- switch from hard-coded values in assets/js/ui-extra.js:
  from "Cancel" to "common:actions.cancel"
- switch from hard-coding by introducing localization keys for the values, e.g.:
```
  "common:actions.select": "Select",
  "common:newPageModal.virtualFolders": "Virtual Folders",
  "common:newPageModal.newPageLocation": "Select New Page Location",
  "common:newPageModal.pages": "Pages",
  "common:newPageModal.newPagePath": "new-page".
```

**Host Info:**
 - OS: CentOS Linux release 8.2.2004 (Core)
 - Wiki.js version: 2.5.136
 - Database engine: PostgreSQL 12.1

**Additional context**
Tested with Croatian Base locale, verified in the corresponding assets JS file.
Bug is not specific to the latest version.
There are other terms that are not localised as well, but they are under Administration area hence they do not impact UX of a regular user. It would be great to have wiki completely localisable, though.

**Comment by NGPixel** [comment]
Fixed by 741b99098af8afb2fc8edc74187c5a9e5ee50fad

**Comment by madodig** [comment]
Thanks @NGPixel!
